From: Ilya Lipnitskiy <ilya.lipnitskiy@gmail.com>
Subject: [PATCH] libfakeroot.c: define _STAT_VER if not already defined

glibc 2.33 no longer declares _STAT_VER. Backported from fakeroot commit
feda578ca3608b7fc9a28a3a91293611c0ef47b7.

diff --git a/libfakeroot.c b/libfakeroot.c
--- a/libfakeroot.c
+++ b/libfakeroot.c
@@ -90,6 +90,16 @@
 #define SEND_GET_XATTR64(a,b,c) send_get_xattr64(a,b)
 #endif
 
+#ifndef _STAT_VER
+ #if defined (__aarch64__)
+  #define _STAT_VER 0
+ #elif defined (__x86_64__)
+  #define _STAT_VER 1
+ #else
+  #define _STAT_VER 3
+ #endif
+#endif
+
 /*
    These INT_* (which stands for internal) macros should always be used when
    the fakeroot library owns the storage of the stat variable.
